config.json

```diff
@@ -275,6 +281,12 @@
             "ConfigurationGroup": "Release"
           }
         },
+        "allConfigurations": {
```

Comment:
I have another change I'm working on where I use "all" to mean "build.cmd" and "build-tests.cmd" This may cause confusion. Perhaps I will change my to "includeTests" or something.